pylint/lint.py

```diff
@@ -814,7 +814,13 @@ def process_tokens(self, tokens):
             match = utils.OPTION_RGX.search(content)
             if match is None:
                 continue
-
+            # Find the Pylint comment among all other special comments on the line
```

Comment:
This doesn't make sense, `OPTION_RGX` should be enough to find the corresponding pragma control. It also assumes that we only have `disable`, which is not the case.